tests/containers/host_configuration.pm

```diff
@@ -27,6 +27,8 @@ sub run {
     else {
         assert_script_run "dhclient -v";
         assert_script_run "curl http://ca.suse.de/certificates/ca/SUSE_Trust_Root.crt -o /etc/ssl/certs/SUSE_Trust_Root.crt";
+        # Stop unattended-upgrades on Ubuntu host to prevent lock processes from automatic updates
+        assert_script_run "sed -i 's/1/0/g' /etc/apt/apt.conf.d/20auto-upgrades " if check_os_release("ubuntu", "PRETTY_NAME");
```

Comment:
The sed command seems overly generic. There's also a superfluous space.

Reply:
i made it less generic and extra space removed